### PR TITLE
removing out of date example ansible.cfg

### PR DESCRIPTION
**Pull request by bcoca**
now has instructions on how to generate up date one via ansible-config

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

examples/ansible.cfg

### CODE REVIEW

**Comment by bcoca** [comment]
!rebuild_merge

File: examples/ansible.cfg [hunk 1]
```diff
@@ -1,525 +1,5 @@
-# Example config file for ansible -- https://ansible.com/
-# =======================================================
+# We removed this file as it was not kept current, but now you can generate it, use:
+# ansible-config init --disabled > ansible.cfg
```
Comment (acozine) [inline examples/ansible.cfg]: How about this?
```suggestion
# To generate an example config file (a "disabled" one with all default settings, commented out):
#               $ ansible-config init --disabled > ansible.cfg
```

**Review by acozine — COMMENTED** [review]
It's a good idea to remove the usually-stale example config file and add instructions for generating one. I put a suggestion on that I hope will help users understand the command and the expected output.

**Comment by satishchhatpar** [comment]
I tried executing below
./ansible-config init --disabled > ansible.cfg

I get this error

usage: ansible-config [-h] [--version] [-v] {list,dump,view} ...
ansible-config: error: argument action: invalid choice: 'init' (choose from 'list', 'dump', 'view')


Versions
ansible-config [core 2.11.4] 
python version = 3.7.0

**Comment by tgvaughan** [comment]
The [current documentation](https://docs.ansible.com/ansible/latest/installation_guide/intro_configuration.html) directly links to this file though.

**Comment by bcoca** [comment]
it should really link to the specific branch, not devel, in general, not just this file, I'll add a 'fix' for this

**Comment by bcoca** [comment]
should be fixed with #75629 #75630 #75631

looking at this more globally as we might have many other /devel/ links that should be /<myversion>/ instead

**Pull request merged**